### PR TITLE
updating easybuild

### PR DESCRIPTION
**Pull request by jerowe**
Updating easybuild packages.

Ping @boegel.

### CODE REVIEW

**Comment by jerowe** [comment]
@boegel 

I required setuptools in the vsc-install meta. Do you know what this madness is?

```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=git+https://github.com/hpcugent/vsc-install.git#egg=vsc-install-0.10.1
```

**Comment by jerowe** [comment]
@scopatz   - Merge?

**Comment by scopatz** [comment]
Fails on windows, but windows is skipped. @conda-forge/staged-recipes, is this safe to merge?

File: recipes/easybuild-easyblocks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package:
+  name: easybuild-easyblocks
+  version: "3.3.1"
+
+source:
+  fn: easybuild-easyblocks-3.3.1.tar.gz
+  url: https://pypi.python.org/packages/7c/03/0ca88b299508689eea650652f5188f9f03d8fc3001670ad37885e71b6e4f/easybuild-easyblocks-3.3.1.tar.gz
+  md5: b53fc64807b56a8bef94b26ae5c5b1a6
```
Comment (isuruf) [inline recipes/easybuild-easyblocks/meta.yaml]: See the example recipe for how the url should be
Use a jinja variable for the version
Also we prefer sha256

**Comment by isuruf** [comment]
For updating recipes please send PRs to the relevant feedstock. For example, https://github.com/conda-forge/vsc-base-feedstock

**Comment by jerowe** [comment]
These are all updates, so I will open PRs. 

How does that work? Do I overwrite the existing recipes?

**Comment by isuruf** [comment]
You have to fork the repo, git clone the repo locally, make changes, commit, push to the fork, open a PR. See https://github.com/conda-forge/vsc-base-feedstock#updating-vsc-base-feedstock

**Pull request closed without merging**